### ISSUE TITLE
Book Appointment Form Redesigning 

### ISSUE THREAD

**Issue by ShreyaDhiman24** [AbhiPatel10/AISKCON-CONSTRUCTION#329]
I want to work on this form redesigning under GSSoC'23

**Comment by Neilblaze** [comment]
@ShreyaDhiman24 I'd rather suggest dropping suggestions (such as low-level prototype designs) on how you're willing to bring changes to the current design. Once it gets approved, you'll be assigned & then can start working on it.

**Comment by ShreyaDhiman24** [comment]
Like sepearting sections , adding 
form image, aligning submit button

**Comment by Neilblaze** [comment]
would you mind pasting some visual elements for reference?

**Comment by ShreyaDhiman24** [comment]
Yeah sure, but don't you think I have to work on that to provide you with visual elements..??

**Comment by Neilblaze** [comment]
@ShreyaDhiman24 Basic wireframes should work

**Comment by ShreyaDhiman24** [comment]
Okay sure i will update you with that..

**Comment by ShreyaDhiman24** [comment]
![image](https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/assets/98320971/4a314a32-b905-4d53-b71e-d6368fd9d05e)

**Comment by ShreyaDhiman24** [comment]
What ?

**Comment by Neilblaze** [comment]
@ShreyaDhiman24 don't worry. I've tagged & awaiting a positive response from @AbhiPatel10

**Comment by ShreyaDhiman24** [comment]
Okay, I will be waiting.
